apps/crn-frontend/src/discover/tutorials/Routes.tsx

```diff
@@ -0,0 +1,27 @@
+import { FC, lazy, useEffect } from 'react';
+import { Switch, Route, useRouteMatch } from 'react-router-dom';
+import { tutorials } from '@asap-hub/routing';
+import { Frame } from '@asap-hub/frontend-utils';
+
+const loadTutorial = () =>
+  import(/* webpackChunkName: "tutorials-details-page" */ './Tutorial');
+const TutorialDetailsPage = lazy(loadTutorial);
+
+const Tutorial: FC<Record<string, never>> = () => {
+  const { path } = useRouteMatch();
+  useEffect(() => {
+    loadTutorial();
+  });
+
+  return (
+    <Switch>
```

Comment:
You don't need a switch for one route however I think you are missing a 404 page when an ID isn't recognized